src/main/java/mate/jdbc/dao/ManufacturerDao.java

```diff
@@ -0,0 +1,16 @@
+package mate.jdbc.dao;
+
+import java.util.List;
+import mate.jdbc.model.Manufacturer;
+
+public interface ManufacturerDao {
+    Manufacturer create(Manufacturer manufacturer);
+
+    Manufacturer get(Long id);
```

Comment:
You need to return Optional<Manufacturer> in this method. Check [task description](https://github.com/mate-academy/JDBC-test-task?tab=readme-ov-file#manufacturerdao-methods) and refactor your implementation.
All in all it's almost always better practice to use Optional when you are not sure that you'll recieve an Entity from any method.
```suggestion
    Optional<Manufacturer> get(Long id);
```